en/04_Changelogs/5.2.0.md

```diff
@@ -12,6 +12,7 @@ title: 5.2.0 (unreleased)
   - [Create random passwords for new users](#create-random-passwords-for-new-users)
   - [Buttons to select all files and deselect all files](#bulk-action-buttons)
   - [More nuanced permissions for `/dev/*` routes](#dev-route-permissions)
+  - [React forms now throw an exception if a field type doesn't have a react component](#react-forms-exception)
```

Comment:
```suggestion
  - [New exception in React forms](#react-forms-exception)
```
Update the heading below as well. You can change the wording but it should be short. The heading doesn't need to convey all the information :p

Reply:
Updated